src/test/java/lotto/domain/LottoMachineTest.java

```diff
@@ -0,0 +1,74 @@
+package lotto.domain;
+
+import lotto.domain.purchaseStrategy.AutoPurchaseStrategy;
+import lotto.domain.purchaseStrategy.PurchaseStrategy;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class LottoMachineTest {
+
+    private LottoMachine lottoMachine;
+    private PurchaseStrategy purchaseStrategy = new AutoPurchaseStrategy();
+
+    @BeforeEach
+    void setUp() {
+        lottoMachine = new LottoMachine();
+    }
+
+    @DisplayName("지불금액이 천원 이하일때 예외 throw")
+    @Test
+    void buyLotto_1000원_이하() {
```

Comment:
한글로 메서드명을 짓는것을 의도하였다면 클래스에 @SuppressWarnings("NonAsciiCharacters")를 적용하여 IDE의 경고를 제거해보시는게 어떨까요?

Reply:
아래와 같이 변경을 해보았는데, 혹시 어너테이션을 붙히는 순서도 따로 컨벤션이 있을까요?
```suggestion
    @DisplayName("지불금액이 천원 이하일때 예외 throw")
    @SuppressWarnings("NonAsciiCharacters")
    @Test
    void buyLotto_1000원_이하() {
```

Reply:
애노테이션을 붙이는 컨벤션은 따로 존재하진 않을 것 같은데요
개인적으론 연관도가 높은 것들을 모아두고 우선순위가 높은 것을 아래로 위치시키고 낮은 것을 위로 위치시킵니다 :)